spring-core/src/test/java/org/springframework/util/comparator/BooleanComparatorTests.java

```diff
@@ -29,34 +29,35 @@
  * @author Keith Donald
  * @author Chris Beams
  * @author Phillip Webb
+ * @author Eugene Rabii
  */
 class BooleanComparatorTests {
 
```

Comment:
tests have changed because the documentation of `Boolean::compare` does not guarantee `-1, 0, 1`; but it does less than 1, 0, greater than 1